Xamarin.Forms.Core/PlatformConfiguration/iOSSpecific/Page.cs

```diff
@@ -56,5 +56,82 @@ public static UIStatusBarAnimation PreferredStatusBarUpdateAnimation(this IPlatf
 			SetPreferredStatusBarUpdateAnimation(config.Element, value);
 			return config;
 		}
+
+		public static readonly BindableProperty UseSafeAreaProperty = BindableProperty.Create(nameof(UseSafeArea), typeof(bool), typeof(Page), false);
+
+		public static bool GetUseSafeArea(BindableObject element)
+		{
+			return (bool)element.GetValue(UseSafeAreaProperty);
+		}
+
+		public static void SetUseSafeArea(BindableObject element, bool value)
+		{
+			element.SetValue(UseSafeAreaProperty, value);
+		}
+
+		public static IPlatformElementConfiguration<iOS, FormsElement> SetUseSafeArea(this IPlatformElementConfiguration<iOS, FormsElement> config, bool value)
+		{
+			SetUseSafeArea(config.Element, value);
+			return config;
+		}
+
+		public static bool UseSafeArea(this IPlatformElementConfiguration<iOS, FormsElement> config)
```

Comment:
Could we consider a different name here that doesn't sound like an action if it doesn't do something? I believe it's just a getter. UseBlurEffect does something for example. Perhaps UsingSafeArea?